packages/core/src/utils/spanUtils.ts

```diff
@@ -85,12 +85,10 @@ function ensureTimestampInSeconds(timestamp: number): number {
 
 /**
  * Convert a span to a JSON representation.
- * Note that all fields returned here are optional and need to be guarded against.
- *
- * Note: Because of this, we currently have a circular type dependency (which we opted out of in package.json).
- * This is not avoidable as we need `spanToJSON` in `spanUtils.ts`, which in turn is needed by `span.ts` for backwards compatibility.
- * And `spanToJSON` needs the Span class from `span.ts` to check here.
  */
+// Note: Because of this, we currently have a circular type dependency (which we opted out of in package.json).
+// This is not avoidable as we need `spanToJSON` in `spanUtils.ts`, which in turn is needed by `span.ts` for backwards compatibility.
+// And `spanToJSON` needs the Span class from `span.ts` to check here.
```

Comment:
I stumbled onto this but found that this comment had no place in JSDoc.

Reply:
> Note that all fields returned here are optional and need to be guarded against.

I'd argue this might still be a helpful message but the types should hint that anyway.
Also just to double-check: Putting a line comment between the JSDoc block and the function declaration doesn't prevent the JSDoc from being associated with the function, right?

Reply:
Yep the JSdoc still works!